lms/envs/common.py

```diff
@@ -2967,4 +2967,7 @@
 REDIRECT_CACHE_KEY_PREFIX = 'redirects'
 #####RACCOONGANG########
 DOC_LINK_BASE_URL = ''
+
+# JIVOSITE widget settings:
+JIVOSITE_WIDGET_ID = os.environ.get('JIVOSITE_WIDGET_ID')
```

Comment:
@wowkalucky also you can read from json without os.environ: as an example: https://github.com/raccoongang/edx-platform/blob/6f4fc73d639751c0d596374a051ced81bf90f267/lms/envs/aws.py#L256